packages/odata-service-inquirer/test/unit/prompts/cap-project/cap-helper.test.ts

```diff
@@ -138,6 +145,73 @@ describe('cap-helper', () => {
         expect(findCapProjectsSpy).toHaveBeenCalledWith({ 'wsFolders': ['/test/mock/'] });
     });
 
+    test('getCapProjectChoices (Windows paths)', async () => {
+        const pathSep = '\\'; // Windows path separator
+        const drive = 'C:'; // Windows drive letter
+
+        // Zero state test
+        const findCapProjectsSpy = jest.spyOn(sapuxProjectAccess, 'findCapProjects').mockResolvedValue([]);
```

Comment:
We should be running a different test on Windows and Linux. i.e on Windows check that realpath is called. On Linux it should not be called.